tests/functional/event_handler/test_bedrock_agent.py

```diff
@@ -0,0 +1,79 @@
+import json
+from typing import Any, Dict
+
+from aws_lambda_powertools.event_handler import BedrockAgentResolver, Response, content_types
+from aws_lambda_powertools.utilities.data_classes import BedrockAgentEvent
+from tests.functional.utils import load_event
+
+claims_response = "You have 3 claims"
+
+
+def test_bedrock_agent_event():
+    # GIVEN a Bedrock Agent event
+    app = BedrockAgentResolver()
+
+    @app.get("/claims")
```

Comment:
for docs later... we need to document that all options will be ignored for Bedrock.